htdocs/product/fournisseurs.php

```diff
@@ -924,6 +987,23 @@ function update_price_from_multicurrency() {
 						print dol_print_date(($productfourn->fourn_date_modification ? $productfourn->fourn_date_modification : $productfourn->date_modification), "dayhour");
 						print '</td>';
 
+						// Extrafields
+						$resql = $db->query("SELECT * FROM " . MAIN_DB_PREFIX . "product_fournisseur_price_extrafields WHERE fk_object = " . $productfourn->product_fourn_price_id);
+						if ($db->num_rows($resql) != 1) {
+							foreach ($extralabels as $key => $value) {
```

Comment:
$extralabels is deprecated.
You must use $extrafields->attributes["product_fournisseur_price"]['label'] instead.